### ISSUE TITLE
create_openstudio_standards_space_data_json_ashrae_90_1() got an unexpected keyword argument 'osstd_repository_path'

### ISSUE THREAD

**Issue by jiangyilin123** [pnnl/building-energy-standards-data#88]
QuickStartGuide says the following code block can be used to generate the space type-related data needed by Openstudio-Standards 
```
import sqlite3
conn = sqlite3.connect('openstudio_standards.db')
from building_energy_standards_data.applications.create_openstudio_standards_json import create_openstudio_standards_space_data_json_ashrae_90_1
for t in ["2004", "2007", "2010", "2013", "2016", "2019"]:
    create_openstudio_standards_space_data_json_ashrae_90_1(conn=conn, version_90_1=t, osstd_repository_path="./")
conn.close()
```

I have the following error: 
`TypeError: create_openstudio_standards_space_data_json_ashrae_90_1() got an unexpected keyword argument 'osstd_repository_path'
`

However, `create_openstudio_standards_data_json_ashrae_90_1` can take ` osstd_repository_path` and run.

**Comment by lymereJ** [comment]
@jiangyilin123 - `osstd_repository_path` is a valid argument to `create_openstudio_standards_space_data_json_ashrae_90_1()`. How are you running the code that you shared? From a clone of the repository, or, did you install the package using `pip`?

**Comment by jiangyilin123** [comment]
I ran it through a repository clone (develop branch). Not sure if the following screen snap helps:<img width="1003" alt="create_db" src="https://github.com/user-attachments/assets/b77bdbb9-cf02-4e6e-a327-fb761f956706">

**Comment by lymereJ** [comment]
I see, thanks! We'll add that shortly.

**Comment by jiangyilin123** [comment]
Hi @lymereJ, I saw the `create_openstudio_standards_data_json_ashrae_90_1` been changed, and now take `osstd_repository_path` as an argument. But this seems not updated to the `QuickStart.md` . 
<img width="842" alt="Issue" src="https://github.com/user-attachments/assets/3b4c9c81-9600-4579-878a-075d98f0f86c">

**Comment by lymereJ** [comment]
Thanks! Addressed in #96.
